src/Algebra/Morphism/Construct/DirectProduct.agda

```diff
@@ -59,13 +58,34 @@ module Magma (M : RawMagma a ℓ₁) (N : RawMagma b ℓ₂) where
                           IsMagmaHomomorphism P N h →
                           IsMagmaHomomorphism P (rawMagma M N) (Product.< f , h >)
     isMagmaHomomorphism F H = record
-      { isRelHomomorphism = < F.isRelHomomorphism , H.isRelHomomorphism >
+      { isRelHomomorphism = RP.< F.isRelHomomorphism , H.isRelHomomorphism >
       ; homo              = λ x y → F.homo x y , H.homo x y
       }
       where
         module F = IsMagmaHomomorphism F
         module H = IsMagmaHomomorphism H
 
+-- Package for export
+module Magma-Export {M : RawMagma a ℓ₁} {N : RawMagma b ℓ₂} where
+  open Magma
+
+  private
+    module M = RawMagma M
+    module N = RawMagma N
+
+  module _ {refl : Reflexive M._≈_} where
```

Comment:
why use an anonymous module here, and no explicit signature? This feels like it is making things obscure.

Reply:
I followed the approach in #2720 (for consistency). Meaning, that implicit parameters are wrapped in anonymous modules

Reply:
I wish I had noticed that in that PR! But I can't fault you for being consistent.

Reply:
And #2720 is yet to be merged, so feel free @JacquesCarette to comment there on my design (which you had previously approved?)